doc/reverse_engineering/game_mechanics/swgb/stealth.md

```diff
@@ -0,0 +1,33 @@
+# Stealth mechanics in SWGB
```

Comment:
Wouldn't it be better to merge the information from `detection.md` into this file?

Reply:
Completely merge the two files? Yeah, that's doable, I hesitated a bit.

Reply:
It should be fine given that the two mechanics are connected.